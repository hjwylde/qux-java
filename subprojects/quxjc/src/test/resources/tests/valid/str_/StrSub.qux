void main([str] args):
    print '012345' - '45'
    print '0123' - '4'

