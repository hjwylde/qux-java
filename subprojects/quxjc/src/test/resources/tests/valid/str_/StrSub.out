0123
0123
